frontend/packages/core/src/Cards/workflow-card.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from "react";
```

Comment:
no changes made here other than renaming it to `WorkflowCard`

Reply:
Should we include this in the dash package instead of core?

Reply:
moved to projectSelector pkg. 

side note: was going to update pkg name to dash for clarity but realized that would cause conflicts in the internal repo as the pkg there is also called dash